### ISSUE TITLE
PR #98 merged but missing files

### ISSUE THREAD

**Issue by sblendorio** [bes-dev/stable_diffusion.openvino#114]
I see that PR #98 has been merged but I don't see neither docker files, nor README.md lines I added...

**Comment by bes-dev** [comment]
@sblendorio yes, we removed some updates temporarily due to incorrect work in our test environment. We will return it back soon.
